src/types/undo-redo/undoRedoManager.ts

```diff
@@ -40,6 +42,7 @@ export class UndoRedoManager {
     this.notifyListeners();
     console.log(this.redoBuf);
     console.log(this.undoBuf);
+    deselectElement();
```

Comment:
I think this is needed only when the undo/redo target is selected, and the target would be removed by the action. We can revisit this later